examples/siro_sandbox/controllers/gui_controller.py

```diff
@@ -180,10 +180,16 @@ def __init__(
         gui_input,
         env,
         walk_pose_path,
+        lin_speed,
+        ang_speed,
         recorder,
     ):
         super().__init__(agent_idx, is_multi_agent, gui_input)
-        self._humanoid_controller = HumanoidRearrangeController(walk_pose_path)
+        humanoid_controller = HumanoidRearrangeController(walk_pose_path)
```

Comment:
Inspired by [this code](https://github.com/facebookresearch/habitat-lab/blob/8967967ba85e03f1d6653c00e2a4d17f492064bf/habitat-lab/habitat/tasks/rearrange/actions/oracle_nav_action.py#L90).